### PR TITLE
feat: add exchangerate api

### PR DESCRIPTION
**Pull request by rahmat-st**

### CODE REVIEW

File: src/content/apis/exchange-rate/exchangerate-api.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: ExchangeRate-API
+description: The Accurate & Reliable Exchange Rate API
+categories:
+  - exchange-rate
```
Comment (TheLearneer) [inline src/content/apis/exchange-rate/exchangerate-api.md:5]: I believe exchange-rate is too specific, I believe it's better to have a specific category like currency
```suggestion
  - currency
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: src/content/apis/exchange-rate/exchangerate-api.md (hunk 1 above)
Comment (rahmat-st) [reply src/content/apis/exchange-rate/exchangerate-api.md:5]: do I have to change the name of the folder to currency also?

Comment (TheLearneer) [reply src/content/apis/exchange-rate/exchangerate-api.md:5]: Yes please

Comment (rahmat-st) [reply src/content/apis/exchange-rate/exchangerate-api.md]: Done

**Review by TheLearneer — APPROVED** [review]
Thank you very much for your contribution @rahmat-st 🥳

File: src/content/apis/currency/exchangerate-api.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: ExchangeRate-API
```
Comment (TheLearneer) [inline src/content/apis/currency/exchangerate-api.md:2]:
```suggestion
name: ExchangeRate API
```

**Pull request merged**